### ISSUE TITLE
Tiny edits on Learn More page

### ISSUE THREAD

**Issue by ashlchap** [reskillamericans/website#71]
Under Phase I button:
--Web Development, 2nd paragraph, 2nd sentence: add a period after confidence:
 As before, we rely heavily on (fun) projects along the way to help cement concepts and build your confidence. 

--Product Design, last sentence -- insert comma after tool: 
You will be trained to use Figma, the industry standard interface-design tool, to communicate your designs to stakeholders and developers. 

Under Job Prep button:
--Capitalize the headline (to be consistent with others): 
Getting Your First Job or Internship

**Comment by mckoss** [comment]
The Phase 1 Product Design sentence on the current site doesn't match what you had.  I see:

> You will have extensive practice using the user-interface design app, Figma, to communicate
your designs to stakeholders and developers.

I've made the other edits.

**Comment by ashlchap** [comment]
Mike, my mistake, so sorry: I just realized that sentence I asked you to edit is actually located on the homepage. It's below for reference, BUT THAT being said, we got good feedback from Amanda about making the homepage copy more brief. If you're okay with the look of it, I'm going to revise the copy on those two track buttons on homepage to reduce text and try to bullet point. So ignore the below, it was just FYI, I'll post something new in the issue tracker.
*******
--Product Design, last sentence -- insert comma after tool:
You will be trained to use Figma, the industry standard interface-design tool, to communicate your designs to stakeholders and developers.
